src/app/mini-games/pages/mini-games-page/mini-games-page.component.scss

```diff
@@ -0,0 +1,66 @@
+:host {
+    position: relative;
+    background: #344557
+        url(https://unsplash.imgix.net/photo-1423683249427-8ca22bd873e0?fit=crop&fm=jpg&h=700&q=75&w=1050) 0 0 no-repeat;
+    background-size: cover;
+    min-height: 85vh;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-wrap: wrap;
+
+    &::before {
+        content: "";
+        position: absolute;
+        top: 0;
+        left: 0;
+        width: 100%;
+        height: 100%;
+        margin: auto;
+        background: rgba(#344557, 0.85);
+    }
+
+    .flex-row {
+        display: flex;
+        justify-content: center;
+        flex-wrap: wrap;
+
+        .example-card {
```

Comment:
@suggestion

It would be better to rename ```example-card``` class

Reply:
done